packages/cli/lib/lib/webpack/run-webpack.js

```diff
@@ -46,7 +46,7 @@ async function devBuild(env) {
 			let serverAddr = `${protocol}://${host}:${bold(port)}`;
 			let localIpAddr = `${protocol}://${ip.address()}:${bold(port)}`;
 
-			clear(true);
+			if (config['clear']) clear(true);
```

Comment:
Here `config` is the webpack config - we want the CLI argv which comes down as `env` here.
```suggestion
			if (env['clear']) clear(true);
```